modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-editor/src/main/resources/META-INF/resources/editor.soy

```diff
@@ -16,9 +16,9 @@
 	{@param placeholder: string}
 	{@param required: bool}
 	{@param showLabel: bool}
-	{@param? tip: string}
 	{@param value: string}
 	{@param visible: bool}
+	{@param? tip: string}
```

Comment:
Hey @clovisdasilvaneto , could you put these declarations in alphabetical order?

Reply:
@diegonvs according the check_sf, optional parameters should be at the end